modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -166,6 +166,7 @@ conf = {{{packageName}}}.Configuration(
 
     def __init__(self, host=None,
                  api_key=None, api_key_prefix=None,
+                 access_token=None,
```

Comment:
Not all specs will use this access token argument.
How about only adding this parameter to this signature if 
{{#hasOAuthMethods}}
or {{#hasBearerMethods}}?

Reply:
Actually that's a nice to have for the future for all of our specific auth inputs. api_key, api_key_prefix, username, password.

Reply:
Ah, yes that makes sense. As for adding the `hasOAuthMethods` check, is there anything else missing in addition to that for this PR ?